code/game/turfs/open/floor/iron_floor.dm

```diff
@@ -295,6 +295,9 @@
 	base_icon_state = "white_herringbone"
 	floor_tile = /obj/item/stack/tile/iron/white/herringbone
 
+/turf/open/floor/iron/white/textured_large/airless //SR Edit - Airless Version
```

Comment:
Put this in its own file

Reply:
have a PR up to add airless version to all the iron tiles in TG - i'll just make sure this is removed